spec/controllers/admin/posts_controller_spec.rb

```diff
@@ -0,0 +1,97 @@
+
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.